docs/api/window-open.md

```diff
@@ -1,5 +1,7 @@
 # The `window.open` function
 
+> Create a new window in a web page.
```

Comment:
`in a web page` sounds a bit weird here.

Perhaps `Open a new window with and load a URL`?